websites/Y/Yandex/presence.ts

```diff
@@ -0,0 +1,126 @@
+const presence = new Presence({
+		clientId: "1035433850593095740",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	homepageInput =
+		document.querySelector<HTMLInputElement>('input[name="text"]'),
+	{ host, pathname } = document.location;
+
+let strings: Awaited<ReturnType<typeof getStrings>>;
+
+enum Assets {
+	logo = "https://i.imgur.com/SFXKtLF.png",
+	search = "https://i.imgur.com/wYVlwJX.png",
+	view = "https://i.imgur.com/hxvvGUi.png",
+}
```

Comment:
enum properties should be Title case

Reply:
> enum properties should be Title case

I don't quite understand what the problem is. Enum properties must be capitalized?

Reply:
> > enum properties should be Title case
> 
> I don't quite understand what the problem is. Enum properties must be capitalized?

Basically, Enum's generally follow the Title case standard. So the first letter is capitalised.